File: examples/android/CMeSdkJava/app/src/main/java/me/connect/sdk/java/sample/db/entity/Connection.java
package me.connect.sdk.java.sample.db.entity;

import androidx.room.ColumnInfo;
import androidx.room.Entity;
import androidx.room.PrimaryKey;

@Entity
public class Connection {
    @PrimaryKey(autoGenerate = true)
    public int id;

    @ColumnInfo(name = "serialized")
    public String serialized;

    @ColumnInfo(name = "name")
    public String name;

    @ColumnInfo(name = "icon")
    public String icon;

    @ColumnInfo(name = "pwdid")
    public String pwDid;

    @ColumnInfo(name = "invitation")
    public String invitation;
}

File: Examples/android/CMeSdkJava/app_kt/src/main/kotlin/me/connect/sdk/java/samplekt/db/entity/Connection.kt
package me.connect.sdk.java.samplekt.db.entity

import androidx.room.ColumnInfo
import androidx.room.Entity

import androidx.room.PrimaryKey


@Entity
data class Connection(
        @PrimaryKey(autoGenerate = true)
        var id: Int = 0,

        @ColumnInfo(name = "serialized")
        var serialized: String,

        @ColumnInfo(name = "name")
        var name: String,

        @ColumnInfo(name = "icon")
        var icon: String?
)
